### PR TITLE
Update lambda

### PR DESCRIPTION
**Pull request by tyler-wenzel**
A new lambda was added to the lambda.tf file for the update user functionality. This addition required us to also add another route to the api_gateway to route requests to the appropriate lambda. Redshift values were refactored to allow for them to be accessed in the "update lambda" to establish connection with database.

### CODE REVIEW

File: redshift.tf [hunk 1]
```diff
@@ -133,6 +133,7 @@ resource "aws_redshift_cluster" "redshift_cluster" {
   provisioner "local-exec" {
```
Comment (brandoncorey) [inline redshift.tf]: This does not need to be here anymore as the tables are created via the startup script for the EC2 instance.

**Review by brandoncorey — CHANGES_REQUESTED** [review]
Good work Tyler. Left a comment asking for a revision. Just a small adjustment.

File: redshift.tf (hunk 1 above)
Comment (tyler-wenzel) [reply redshift.tf]: Thanks for calling that out. The local-exec block is now deleted.

**Review by brandoncorey — APPROVED** [review]
LGTM!

**Pull request merged**